pkg/controller/elasticsearch/initcontainer/prepare_fs.go

```diff
@@ -7,15 +7,17 @@ package initcontainer
 import (
 	"path"
 
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
```

Comment:
we discussed offline: it may be related to IntelliJ settings. We agreed that it's OK to have several import groups as long as they make sense (eg. no `github.com/elastic` in different groups).